replay: avoid copy paste mistake without reading docs (#11117)
getsentry/sentry-docs: platform-includes/replay/privacy-configuration/javascript.mdx

```javascript
Sentry.replayIntegration({

>   // NOTE: This will disable built-in masking. Look at other options opt-in to masking sensitive data

Reviewer: Not sure why the second sentence is supposed to mean

Reviewer 2: I would say, set these options as 'true' to mask sensitive data

Reviewer 2: If we really want to add a comment here, I think first sentence will suffice

>   // NOTE: This will disable built-in masking. Look at other options opt-in to masking sensitive data

Reviewer 2: ```suggestion
  // NOTE: This will disable all text and media masking. 
```

>   // NOTE: This will disable built-in masking. Look at other options opt-in to masking sensitive data

Author: ```suggestion
  // NOTE: This will disable built-in masking. Only use this if you site has no sensitive data or, paired with other options for blocking parts of the site with sensitive data such as 'ignore', 'block', 'mask' and 'maskFn'
```
